src/main/proto/commit.proto

```diff
@@ -28,3 +28,16 @@ message Commit {
     int32 num_lines_deleted = 8;
     repeated Stats commit_stats = 9;  // Stats on a commit.
 }
+
+// User information.
+message UserInfo {
+    int32 id = 1;  // User id.
```

Comment:
Why assigns this id? We don't need any numeric user id, because we have a unique username, and we should be using that everywhere.